back-end/aws-cloud9-api/src/main/java/com/serverless/dal/Question.java

```diff
@@ -0,0 +1,255 @@
+    	this.id = body.get("id").asText();
+    	this.type = body.get("type").asText();
+    	this.question = body.get("name").asText();
+    	this.answer = body.get("answer").asInt();
+    	this.recruiter_id = body.get("recruiter_id").asText();
+    	this.language = body.get("language").asInt();
+    	this.form_membership = body.get("form_id").asText();	
+    	
+    }
+    public void setClosedQuestion(JsonNode body) {
+    	this.id = body.get("id").asText();
+    	this.type = body.get("type").asText();
+    	this.question = body.get("name").asText();
+    	this.answer = body.get("answer").asInt();
+    	this.recruiter_id = body.get("recruiter_id").asText();
+    	this.language = body.get("language").asInt();
+    	this.form_membership = body.get("form_id").asText();
+    	this.answer_list = new ArrayList<String>();
+    	ObjectMapper mapper = new ObjectMapper();
+		this.answer_list = mapper.convertValue(body.get("answer_list"), ArrayList.class);	
+    	
+    }
+
+    public void setOpenQuestion(JsonNode body) {
+    	this.id = body.get("id").asText();
+    	this.type = body.get("type").asText();
+    	this.question = body.get("name").asText();
+    	this.recruiter_id = body.get("recruiter_id").asText();
+    	this.language = body.get("language").asInt();
+    	this.form_membership = body.get("form_id").asText();
+    	
+    }
+
+   
+    public void update(JsonNode body) {
+		 logger.info("Call Question::update(" + body + ")");
+		 JsonNode newId = body.get("id");
+		 JsonNode newType = body.get("type");
+		 JsonNode newQuestion = body.get("name");
+		 JsonNode newAnswer = body.get("answer");
+		 JsonNode newRecrutierId = body.get("recruiter_id");
+		 JsonNode newForm_membership = body.get("form_membership");
+		 JsonNode newLanguage = body.get("language");
+		 JsonNode newAnswer_list = body.get("answer_list");
+
+		 
+		 
+		 
+		 if(newId != null) {
+			 this.setId(newId.asText());
+		 }
+		 else {
+			 logger.warn("Id is null");
+		 }
+		 
+		 if(newType != null) {
+			 this.setType(newType.asText());
+		 }
+		 else {
+			 logger.warn("newType is null");
+		 }
+		 
+		 if(newQuestion != null) {
+			 this.setQuestion((newQuestion.asText()));
+		 }
+		 else {
+			 logger.warn("Quesrtion is null");
+		 }
+		 if(newRecrutierId != null) {
+			 this.setRecruiter_id((newRecrutierId.asText()));
+		 }
+		 else {
+			 logger.warn("recruiterid is null");
+		 }
+		 if(newForm_membership != null) {
+			 this.setForm_membership((newForm_membership.asText()));
+		 }
+		 else {
+			 logger.warn("membership is null");
+		 }
+		 if(newLanguage != null) {
+			 this.setLanguage((newLanguage.asInt()));
+		 }
+		 else {
+			 logger.warn("Language is null");
+		 }
+		 if(newAnswer != null) {
+			 this.setAnswer((newAnswer.asInt()));
+		 }
+		 else {
+			 logger.warn("Answer is null");
+		 }
+		 if(newAnswer_list != null) {
+			 this.setAnswer((newAnswer_list.asInt()));
+		 }
+		 else {
+			 logger.warn("Answer is null");
+		 }
+	 }
```

Comment:
Ta funkcja jest za długa, nie może tak zostać bo nie spełnia wymagań kodu. Jeśli zmodyfikowanie metody set nie działa, to trzeba wymyślić inny sposób.